docs/operations.md

```diff
@@ -154,6 +154,38 @@ Link it into discovery directory:
 $ sudo ln -s /dev/disk/by-id/lvm-pv-uuid-yyTnct-TpUS-U93g-JoFs-6seh-Yy29-Dn6Irf /mnt/disks
 ```
 
+### Link devices into directory to be discovered as Filesystem PVs
```

Comment:
Can you add this to the TOC above?

Reply:
@mattymo you can generate it with [`markdown-toc`](https://github.com/jonschlinkert/markdown-toc), e.g. `markdown-toc --no-firsth1 docs/operations.md`

Reply:
Thanks for reminding me to do this.